pkg/api/v1beta3/dynakube/validation/modules.go

```diff
@@ -0,0 +1,46 @@
+package validation
+
+import (
+	"context"
+
+	"github.com/Dynatrace/dynatrace-operator/pkg/api/v1beta3/dynakube"
+)
+
+const (
+	errorOneAgentModuleDisabled   = `OneAgent features has been disabled during Operator install. The necessary resources for deploying a OneAgent DaemonSet to work are not present on the cluster. Redeploy the Operator with all the necessary resources`
```

Comment:
can we reuse the constant and only change the component name? Would be easier to update the wording then.

Reply:
I prefer the way you phrased it for Edgeconnect:
> EdgeConnect has been disabled during Operator install. The necessary resources for EdgeConnect to work are not present on the cluster. Redeploy the Operator with all the necessary resources

No mention of features or workload necessary I think

Reply:
Should we mention that it should be done in the `values.yaml`? Or do we trust that everyone knows this?

Reply:
In [Standardize the validation error message](https://github.com/Dynatrace/dynatrace-operator/pull/3853/commits/99409d2d33e6ed576a8200dfd1b1242a73e478e0) I tried to fix this, but depending if this goes in before or after https://github.com/Dynatrace/dynatrace-operator/pull/3856, I might do it differently